src/nunit-gui/Views/MainForm.cs

```diff
@@ -144,5 +144,10 @@ private void statusBarToolStripMenuItem_CheckedChanged(object sender, System.Eve
         {
             statusBarView.Visible = statusBarToolStripMenuItem.Checked;
         }
+
+        protected override void OnDragEnter(DragEventArgs drgevent){
+            if(drgevent.Data.GetDataPresent(DataFormats.FileDrop))
+                drgevent.Effect = DragDropEffects.Copy;
+        }
```

Comment:
Let me double check. That was the only pending change I had and when I built and dragged and dropped it worked on my machine